container.go

```diff
@@ -167,6 +169,14 @@ func (c *container) NewTask(ctx context.Context, ioCreate IOCreation, opts ...Ne
 	return t, nil
 }
 
+func (c *container) Size(ctx context.Context) int {
+	return c.c.Size()
+}
+
+func (c *container) Runtime(ctx context.Context) string {
```

Comment:
Don't need to expose this, either. It is already available on the proto object, which is exposed via `.Proto()`.

I'm not sure I like where this is heading. We shouldn't be exposing the underlying protobufs.